.gitignore

```diff
@@ -104,6 +104,9 @@ db.sqlite3
 # VSCODE
 .vscode/
 
+# Migration data
+authors/apps/articles/migrations/
```

Comment:
I would suggest that its best not to ignore the migrations, and if you have any reasons for, doing it please lets discuss through it